content/docs/1.0.0/advanced-resources/rwx-workloads.md

```diff
@@ -0,0 +1,15 @@
+---
+title: RWX workloads
+weight: 4
+---
+
+{{< figure src="/img/diagrams/rwx/rwx-support.png" >}}
+
+Longhorn supports RWX workloads via the longhorn.io/nfs provisioner.
+To enable RWX support you need to deploy the following files:
+- [01-security.yaml](https://github.com/longhorn/longhorn/examples/rwx/01-security.yaml)
+- [02-longhorn-nfs-provisioner.yaml](https://github.com/longhorn/longhorn/examples/rwx/02-longhorn-nfs-provisioner.yaml)
+
+if you want to run multiple nfs-provisioners you can modify the provisioner argument and
```

Comment:
It would be better to remind users that scaling the deployment is not the correct way to launch multiple nfs-provisioners. Since there should be one unique StorageClass name for a NFS server (if I don't misunderstand `kubernetes_incubator/nfs-provisioner`).